packages/ag-charts-community/src/chart/interaction/regionManager.ts

```diff
@@ -74,14 +75,12 @@ export class RegionManager {
             this.keyNavManager.addListener('submit', this.onNav.bind(this))
         );
 
+        injectStyle(focusStyles.css, focusStyles.block);
         this.focusIndicator = getDocument()?.createElement('div');
         if (this.focusIndicator !== undefined && element !== undefined) {
-            this.focusIndicator.className = 'ag-charts-focusindicator';
-            this.focusIndicator.style.position = 'absolute';
-            this.focusIndicator.style.border = '2px solid red';
-            this.focusIndicator.style.display = 'none';
-            this.focusIndicator.style.pointerEvents = 'none';
-            this.focusIndicator.style.userSelect = 'none';
+            this.focusIndicator.classList.add(focusStyles.block);
+            this.focusIndicator.classList.add(focusStyles.elements.indicator);
```

Comment:
nit: under BEM you'd have elements as a child of the block, not have the block and element on the same node, doesn't really matter but you can probably ditch the element as it doesn't have any styles/behaviour as far as I can see?

Reply:
Fair point, removed.